test/integration/multiplexed_integration_test.cc

```diff
@@ -1817,6 +1817,43 @@ TEST_P(Http2FrameIntegrationTest, UpstreamSettingsMaxStreamsAfterGoAway) {
   tcp_client_->close();
 }
 
+TEST_P(Http2FrameIntegrationTest, UpstreamWindowUpdateAfterGoAway) {
```

Comment:
Can you document a bit more the insight/subtlety of this test ? Thanks

Reply:
Done. There's further context in internal bug b/217743479, FWIW.